includes/helpers/functions/templates-ui.php

```diff
@@ -329,7 +329,7 @@ function wpbdp_main_links( $buttons = null ) {
 
 	if ( in_array( 'create', $buttons, true ) ) {
 		$html[] = '<a href="' . esc_url( wpbdp_url( 'submit_listing' ) ) . '" id="wpbdp-bar-submit-listing-button" class="button wpbdp-button">' .
-			esc_html__( 'Add Listing', 'business-directory-plugin' ) .
+			'+ ' . esc_html__( 'Add Listing', 'business-directory-plugin' ) .
```

Comment:
I'm not loving the way this looks. I think we removed this since the design was created. Can we revert this line too?